### ISSUE TITLE
Saxon/C 1.0.0 is released - moving on from pre-alpha?   Need help?

### ISSUE THREAD

**Issue by danizen** [ajelenak/pysaxon#2]
I just noticed that Saxon/C 1.0.0 is out, and I'm wondering whether you need help getting this up on Pypi and moving on from Alpha.   I thought I might work on this before, but waited because I saw this repository, but you haven't been much active.

**Comment by ajelenak** [comment]
No, I haven't had much time to work on this after the initial effort back in 2016. I would have progressed further then but I hit a strange issue with segfaults because of one C++ global variable in the Saxon/C library that was not needed on the Python side. It may be that my Cython class declarations were not right but nothing jumped out as an obvious and quick fix.
